pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -184,7 +184,7 @@ let
     # loop through all directories in the root
     for dir in /*; do
       # if it is a directory and it is not ignored
-      if [[ -d "$dir" ]] && [[ ! "''${ignored[@]}" =~ "$dir" ]]; then
+      if [[ -d "$dir" ]] && [[ ! "''${ignored[*]}" =~ " $dir " ]]; then
```

Comment:
Wouldn’t this cause the same issue for paths that contain `IFS` (probably spaces in this case), as well as ignore first and last elements of the array? [containsElement](https://stackoverflow.com/a/8574392) function seems more reliable, and also a good candidate for inclusion in stdenv.

```console
$ bash -e -u -c 'declare dir=/foo
  declare -a ignored=(/foo /bar /baz)
  if [[ ! "''${ignored[*]}" =~ " $dir " ]]; then
    echo not ignored, oops
  fi'
not ignored, oops
```

Reply:
I think `"''${etc_ignored[@]}" =~ "$i"` a few lines above should be updated as well.